half-light.js

```diff
@@ -33,7 +33,7 @@
   function refreshTargetedStyles() {
     targetedStyles = [];
     [...document.styleSheets].forEach((sheet) => {
-      if (!sheet.ownerNode.matches("head > *")) return;
+      if (!sheet.ownerNode.matches("head > :not([no-half-light])")) return;
```

Comment:
How much is this actually helping with perf? It just gets filtered out by the MQ anyway right?

Reply:
I mean it just says 'there's nothing more to do on this one" early - otherwise you gotta loop through the whole stylesheet -- what if that is a thing with a few thousand rules and you already know you don't want any of them to apply inside the shadow?

Reply:
Ah yeah that makes sense actually